Blog/Gemfile

```diff
@@ -40,6 +41,8 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails'
+  gem 'rails-controller-testing'
+  gem 'factory_girl_rails'
```

Comment:
`factory_girl` was recently renamed to `factory_bot` You might want to update this at some point.